app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def show
+    # Action to handle https://users/745
+    @user = User.find(params[:user_id])
+  end
+end
```

Comment:
- [ ] Kindly add the with #index action to your user controller as per requirement. t's okay for the action to be empty for now.

e.g
`def index; end`